framework/auth/oauth_scopes.py

```diff
@@ -17,29 +17,29 @@ class CoreScopes(object):
     """
     The smallest units of permission that can be granted- all other scopes are built out of these.
     Each named constant is a single string."""
-    USERS_READ = 'users+read'
-    USERS_WRITE = 'users+write'
+    USERS_READ = 'users_read'
+    USERS_WRITE = 'users_write'
 
-    NODE_BASE_READ = 'nodes.base+read'
-    NODE_BASE_WRITE = 'nodes.base+write'
+    NODE_BASE_READ = 'nodes.base_read'
+    NODE_BASE_WRITE = 'nodes.base_write'
 
-    NODE_CHILDREN_READ = 'nodes.children+read'
-    NODE_CHILDREN_WRITE = 'nodes.children+write'
+    NODE_CHILDREN_READ = 'nodes.children_read'
+    NODE_CHILDREN_WRITE = 'nodes.children_write'
 
-    NODE_CONTRIBUTORS_READ = 'nodes.contributors+read'
-    NODE_CONTRIBUTORS_WRITE = 'nodes.contributors+write'
+    NODE_CONTRIBUTORS_READ = 'nodes.contributors_read'
+    NODE_CONTRIBUTORS_WRITE = 'nodes.contributors_write'
 
-    NODE_FILE_READ = 'nodes.files+read'
-    NODE_FILE_WRITE = 'nodes.files+write'
+    NODE_FILE_READ = 'nodes.files_read'
+    NODE_FILE_WRITE = 'nodes.files_write'
 
-    NODE_LINKS_READ = 'nodes.links+read'
-    NODE_LINKS_WRITE = 'nodes.links+write'
+    NODE_LINKS_READ = 'nodes.links_read'
+    NODE_LINKS_WRITE = 'nodes.links_write'
 
-    NODE_REGISTRATIONS_READ = 'nodes.registrations+read'
-    NODE_REGISTRATIONS_WRITE = 'nodes.registrations+write'
+    NODE_REGISTRATIONS_READ = 'nodes.registrations_read'
+    NODE_REGISTRATIONS_WRITE = 'nodes.registrations_write'
 
-    APPLICATIONS_READ = 'applications+read'
-    APPLICATIONS_WRITE = 'applications+write'
+    APPLICATIONS_READ = 'applications_read'
+    APPLICATIONS_WRITE = 'applications_write'
```

Comment:
I was really trying to separate namespace delimiters (nodes.registrations) from permissions delimiters (registrations_read) in general. Not sure if that design goal makes sense, but...